options/locale/locale_en-US.ini

```
@@ -1512,7 +1512,9 @@ issues.label_description = Description
 issues.label_color = Color
 issues.label_exclusive = Exclusive
 issues.label_archive = Archive Label
-issues.label_archive_tooltip= Archived labels are excluded from the label search when applying labels to an issue. Existing labels on issues remain unaffected, allowing you to retire obsolete labels without losing information.
+issues.label_archived  = Archived
+issues.label_archived_filter = Archived Labels
+issues.label_archive_tooltip = Archived labels are excluded from the label search when applying labels to an issue. Existing labels on issues remain unaffected, allowing you to retire obsolete labels without losing information.
```

The second sentence is unnecessary in this context 
```suggestion
issues.label_archive_tooltip = Archived labels are excluded by default from the suggestions when searching by label.
```